src/translations/sqlb_de.ts

```diff
@@ -3481,12 +3481,12 @@ Sie können SQL-Statements aus der Schemaspalte nehmen und in den SQL-Editor ode
     <message>
         <location filename="../MainWindow.ui" line="1832"/>
         <source>Copy Create statement</source>
-        <translation>Create-Statement kopieren</translation>
+        <translation>Create-Anweisung kopieren</translation>
```

Comment:
If we already are at it, the `CREATE` part should probably use some consistent casing.

Reply:
Ok, thats a good point. 
I changed Create to uppercase, because there are two other lines also in uppercase (line 1990 and 3489)
```suggestion
        <translation>CREATE-Anweisung kopieren</translation>
```

Reply:
Seems like you missed to apply your suggestion?